great_ape_safe/ape_api/ebtc.py

```diff
@@ -1273,12 +1456,14 @@ def close_cdp(self, cdp_id):
         assert cdp_id_status == CdpStatus.CLOSED.value
 
         # 2.2. verify that enough collateral was returned + gas stipend, assertion denominated in common `shares` unit
-        assert self.collateral.getSharesByPooledEth(
-            self.collateral.balanceOf(self.safe.address)
-        ) == (
+        assert approx(
```

Comment:
should not here the fact that we are denominating everything into `shares` mitigate the issue of the 1-2 wei corner case?

Reply:
**https://docs.lido.fi/guides/lido-tokens-integration-guide/#1-2-wei-corner-case**

Reply:
Fixed, thanks!